Userland/Libraries/LibGUI/Scrollbar.h

```diff
@@ -41,6 +41,13 @@ class Scrollbar : public AbstractSlider {
     virtual void change_event(Event&) override;
 
 private:
+    enum class GutterClickState {
+        NotPressed,
+        Before,
+        After,
```

Comment:
Before what? Please name these more clearly (`BeforeScrubber` would probably be fine).